DevVehicle35-Motors/App/CarInteraction.cs

```diff
@@ -47,7 +47,7 @@ private void SelectNumberDoors()
             Console.WriteLine("======================================");
         }
 
-        private string DetermineColor(int optionColor) {
+        private static string DetermineColor(int optionColor) {
             if (optionColor == 1) {
```

Comment:
To maintain good practices based on styleCop and C# we should keep braces on their own lines of code.

Reply:
In line 58, you might compare option color == 4.